js/components/discussions/thread-form.vue

```diff
@@ -22,10 +22,14 @@ export default {
           // Flags to `true` when the form is in sending process. This prevents
           // duplicate POSTing.
           sending: false,
-          comment: ''
+          comment: '',
+          id: null
       }
   },
-  methods: {
+    ready: function() {
+      this.id = this._uid;
```

Comment:
I did not know about `this._uid`, nice find. But have you seen this https://github.com/vuejs/vue/issues/5886#issuecomment-308625735?

I think using `this.discussionId` with a prefix like `new-comment-${this.discussionId}` should be unique enough (and you should be able to use it directly in `data`).

Reply:
This might be simpler indeed